Answers.md

```diff
@@ -1,11 +1,16 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+    It is a JS library that uses components and other backend functionality to create a virtual DOM that makes it possible for websites to be more performant. It was created by facebook to solve the problem of too much data and on each browser change. 
 
-1.  What does it mean to _think_ in react?
+2.  What does it mean to _think_ in react?
+    It means to break things down into components that do one thing, and to put those pieces all together, and it means to create efficient code.
 
-1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+3.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+    Functional components can't have state but class components can. Functional components have less functionality, but they are a good option for efficiency if you don't need state.

```

Comment:
This is good, but be sure you know that  functional components are purely presentational. They have one job, and that's to render content to the DOM, whereas stateful components have to manage the state and the data that is passed down to those functional components.